code/src/components/ClearAll.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import { todoList } from '../reducers/todoList'
+
+const ClearAll = () => {
+  const dispatch = useDispatch()
+
+  return (
+    <Button onClick={() => dispatch(todoList.actions.clearAllTask())}>
+      Clear all
+    </Button>
+    );
+}
+ 
```

Comment:
It's nice that you made this button that marks all tasks as complete. :) But I think the button should have another name, since "clear all" to me implies that you would clear the list (remove all tasks). So a better name for this button I think would be "Mark all complete" or something along those lines.

It would also be nice with a button that removes all tasks from the list. Then you could have in your reducer a function that sets the state.list to empty, and then use it on a button. Just a suggestion :)

Reply:
UPDATE: I actually re-read the project brief that specifies that what you have done with the button here is as it should. So you could ignore my comment about this :)